### DISCUSSION TITLE
Show OG images you made with Satori! ❤️

### DISCUSSION THREAD

**Discussion by shuding** [vercel/satori#173] [Show and tell]
Here you can share your OG images, social cards, or any other images generated with Satori!

_If it's open sourced, feel free to link to the code as well <3_

**Comment by Sam-Apostel** [comment]
repo: https://github.com/Sam-Apostel/confessions
live page with og-image: https://ua.confessions.link/confessions/16010
![og-2](https://user-images.githubusercontent.com/16917762/196062775-dc37c026-095b-4e52-9701-72a0bb1682d9.png)

**Comment by kb1995** [comment]
I created a bunch of templates which you can copy/paste and customise for your project. Hope you find it useful!

https://fullstackheroes.com/resources/vercel-og-templates/

**Reply by shuding** [reply]
Wow thank you for making this!

**Comment by wei** [comment]
Just upgraded my project [socialify](https://github.com/wei/socialify) to use satori and @vercel/og.

[Link](https://socialify.git.ci/vercel/satori?description=1&font=Raleway&forks=1&issues=1&language=1&logo=https%3A%2F%2Fuser-images.githubusercontent.com%2F5880908%2F205455131-a610ecfd-5d4d-4f8a-b500-a58656e56cb2.svg&name=1&owner=1&pattern=Diagonal%20Stripes&pulls=1&stargazers=1&theme=Dark) / Live Preview:
[![satori](https://socialify.git.ci/vercel/satori/image?description=1&font=Raleway&forks=1&issues=1&language=1&logo=https%3A%2F%2Fuser-images.githubusercontent.com%2F5880908%2F205455131-a610ecfd-5d4d-4f8a-b500-a58656e56cb2.svg&name=1&owner=1&pattern=Diagonal%20Stripes&pulls=1&stargazers=1&theme=Dark)](https://socialify.git.ci/vercel/satori?description=1&font=Raleway&forks=1&issues=1&language=1&logo=https%3A%2F%2Fuser-images.githubusercontent.com%2F5880908%2F205455131-a610ecfd-5d4d-4f8a-b500-a58656e56cb2.svg&name=1&owner=1&pattern=Diagonal%20Stripes&pulls=1&stargazers=1&theme=Dark)

**Comment by sahithyandev** [comment]
I am creating OG images for my posts on https://sahithyandev.github.io

![image](https://user-images.githubusercontent.com/29434759/211484651-0318031c-d9df-44ed-b9e0-032c45271d3e.png)

**Comment by shepherdjerred** [comment]
I'm using Satori on the backend to render match reports for League of Legends. I love that I can programatically create images with React + TypeScript + tailwind.

Additionally, I made a [small shim project](https://github.com/shepherdjerred/glitter-boys/tree/main/packages/preview) that allows me to quickly iterate on these images by rendering my component in-browser.

Thanks so much for making this library!

[![](https://raw.githubusercontent.com/shepherdjerred/glitter-boys/main/assets/postmatch.png)](https://github.com/shepherdjerred/glitter-boys/blob/main/assets/postmatch.png)

**Comment by jakeinflab** [comment]
We run **[Inflearn](www.inflearn.com)**, a famous IT knowledge-sharing platform in South Korea. It's similar to Udemy.

We made OG images to preview online course reviews. It seems to encourage our users to share more on social media.

Thank you to everyone contributing to the Satori project.

[Link](https://www.inflearn.com/community/reviews/131523)
![image](https://github.com/vercel/satori/assets/105339343/985b9b07-5d56-4c79-a908-6a8f9cba1e80)

**Comment by ducan-ne** [comment]
I've made https://ogcool.vercel.app og image generator for who want powering blog with a og image without coding, backed by satori
![og](https://github.com/vercel/satori/assets/16853175/3a57947c-66fc-4931-a26b-b9e206c9311f)

it's open source https://github.com/ducan-ne/ogcool

**Comment by kazzyfrog** [comment]
I created a free & open-source LGTM image generator with Satori.

[![LGTM Factory](https://lgtm-factory.pages.dev/api/v1/lgtm-images?theme=iced-coffee&text=Thank+you+%3B%29&emoji=%F0%9F%93%A6&color=%23fcd34d)](https://lgtm-factory.pages.dev)

You can generate your own stickers and copy & paste them on GitHub.
Make your projects fun 👍

web: https://lgtm-factory.pages.dev/
repo: https://github.com/lgtm-factory/lgtm-factory

I really like `satori` and `@vercel/og`.
Thanks so much for making this library !

Have a wonderful day ;)

**Comment by fwwz-id** [comment]
I set up dynamic OG images on my personal site with Satori + Next.js, so every blog post gets its own custom image.

Implementation on NextJs:  
1. [Base function](https://github.com/fwwz-id/personal-website/blob/main/src/lib/og-generator.tsx)
2. [Implementation on opengraph-image.tsx dynamically](https://github.com/fwwz-id/personal-website/blob/main/src/app/blog/%5Bslug%5D/opengraph-image.ts)

[Playground link](https://og-playground.vercel.app/?share=lVZtb9s2EP4rBxUDksB680tje3EwNF2GFc5WIOuGDf5CSZTNliI1korsef7vO8ryiyQH7WzYonhH8rmHzx25dWKZUGfq-Dc3CwE38CgVEIhllnNqKDD7zKgwxDApgAn4ha7NB92rnNFGNAVFU6rASJjCyphcT31_ycyqiDycx0_L8h-XJX5OlZaCcLekkWaG-hGXkZ8RJnytYp-zyJdLd0kFVcRI5Rm9BruKvxALcXUNs3vY4jtiE9qAYYZTmMHCeV6xHP6UhYKn3z9CiQvDE1043598E6pjxfIqAhyB_YDjHpXMgCWUWOTHoTGGJICoeIUQY1Mo2oOU4CQ5WVLdAyIS-Ltg8RdIKU0igg0uZa49-KhIbFhMeA8M1YZEHIdad0VJsrGLaETqNZClUhqkzobxg-XJYwn8Cw84wrAXCu_pC-USiasHWSYA50NYAq72gQDcJezl0AbQZsPpbFtRdfisKFuuzBRXCYPgu4Vjd-_4KVliVq_YEqZzTjbWmnK6bllt13umkCZk1vrEkheZaHkRzpbiZ0MzXbmglmw8DZfPhTYs3TxItIkK50U3y_ZSyUIkD5JLZf1Ku00tt1yivGpEivKKyobLbndo35867_TL8nyWdcaFni0cq2gUdFmWXjnwpFr6_SAIfOvtnPu_MFq-k2scEUAAITrB20HQ9Kmonm2t8QjhtD-zLQ5o9B_2shESiTTSXEWN-agp8hWchdQIyoqDprrRg305MchuJabzD0sQfSKNbqKu2LBxdbs3l7v3cS6c_rhrq2O9bKyBfRLMWO4LTdVzTmL6q_ikMeBX3GvdfNuoFhfIRswUZj3ENsb-woHYRmUbqn6mjHNsqmVEroJe9fXC4TVa_Dazfg2puQd-ZxPubNYcWD3QeOTNCgRfj1RVQjrAKBS_emN3qQ3gzooS3896GpUBwJQ4vnTHftiHjAl3tW_aPK7-XExgyN1wCJFUCRam_cONuK1zp1y6pNEmEZFcP69IIkur2HG-BvsL8FdN1UrY88y2_m_SNG25vK7vVtqexDcctbf-yOcYd7VpambvaASTjktNv5Cio6n_Wysu5OOqLUubi08DCCdeOA8HXn_fGg68CYwCrz8fjo7PPspxPgrRcmqNxt5oPgy9230rDKF_6w3mE298aowm2BgcHnb-v7o5Vodd71vT2pG_DWSPvD9BCuf4j2CC28GIusFb1Ks2Sn6hp_nqDvewabedpOqqeq_rvZgr1YaHA8SNqCkpFbBy04JzKPePhr4XTgd0O-RqZoP3HHe45pAZlxRGdpk5VucU688jyRivDsqq9MBPCk92bZVemf-oT-AJHgvnWr5YkbbV5ablhaWlA_wMKQJ1EaktV6_g-kDNO4W3LQ1PUmA4iKO78tlV6evrt6rLkbqKbzyXM-0ezvFv5O4Cxq9V7_x-u79F7S4g9BFiuxQ3eprv52_XC7Fzeo6suNDOdOtUGnWmtjj3nH0tcaZYmntOQqNi6UxTwjXtOTSTn9lvm9xeq1GO9g0nsoH-mEU0caZGFXTXc_CKiB4ryrkspeKJs_sP)

![](https://fwwz-id.vercel.app/blog/rag-vs-mcp/opengraph-image)

Thank you for making this awesome library! ❤️

**Comment by shepherdjerred** [comment]
I've made these two projects:

- https://github.com/shepherdjerred/scout-for-lol
- https://github.com/shepherdjerred/astro-opengraph-images

**Comment by puppyum** [comment]
**MGuide** ([mguide.app](https://mguide.app)) generates ~350 branded OG images at build time using Satori + Resvg for a University of Michigan campus map.

Two styles depending on whether a building has a photo:
- **Photo variant**: building photo background with gradient overlay, walk time from the Diag, and building name
- **Card fallback**: branded card with UMich blue/maize colors, category icon, and building details

All rendered at 1200×630 via `satori()` → Resvg SVG-to-PNG. The pipeline also generates room-specific OG images for classrooms with 5+ scheduled sections.
